tests/infra/testmode.py

```diff
@@ -0,0 +1,12 @@
+import os
+import unittest
+
+
+def standalone(f):
+    return unittest.skipIf(
+        ("INTEGRATION_TESTS" in os.environ and os.environ['INTEGRATION_TESTS'] != "0"),
+        "Only running standalone tests")(f)
+
+
+def integration(f):
+    return unittest.skipUnless("INTEGRATION_TESTS" in os.environ, "Only running integration tests")(f)
```

Comment:
A test marked as `@integration` will be skipped unless condition is true.
<=> A test marked as `@integration` will be skipped if condition is not true.
<=> A test marked as `@integration` will be skipped if condition is false.
<=> A test marked as `@integration` will be run if condition is true.
<=> A test marked as `@integration` will be run if `IT in env`
=> A test marked as `@integration` will be run if `$IT=0`

That's not correct.

If a test marked as `@integration` is skipped, the message "Only running integration tests" is printed. I don't think that's correct either.

Reply:
Ah, I get your concern.  I totally misread your initial comment. :)
Updated.